### ISSUE TITLE
--filter-tags not working

### ISSUE THREAD

**Issue by kuyagic** [rustic-rs/rustic#1217]
rustic version 0.8.0

My host snapshots below
```
 rustic -P kfr snapshots --filter-host "aless-wang-uk.erl.re" --log-level error

snapshots for (host [aless-wang-uk.erl.re], label [], paths [/opt/docker/app/couchdb.aless])
| ID       | Time                | Host                 | Label | Tags    | Paths                         | Files | Dirs |    Size |
|----------|---------------------|----------------------|-------|---------|-------------------------------|-------|------|---------|
| 94bf99fc | 2024-08-29 11:47:38 | aless-wang-uk.erl.re |       | aless   | /opt/docker/app/couchdb.aless |    13 |   10 | 5.4 GiB |
|          |                     |                      |       | couchdb |                               |       |      |         |
1 snapshot(s)
```

When I use this command to filter snapshots, it shows the unexpected one
```
rustic -P kfr snapshots --filter-tags "aless" --log-level error

snapshots for (host [work-laptop], label [], paths [E:\JetBrains\@proj])
| ID       | Time                | Host        | Label | Tags | Paths              | Files | Dirs |    Size |
|----------|---------------------|-------------|-------|------|--------------------|-------|------|---------|
| 960317dc | 2024-08-14 14:43:11 | work-laptop |       |      | E:\JetBrains\@proj |   559 |   66 | 2.2 MiB |
1 snapshot(s)

total: 1 snapshot(s)
```

**Comment by aawsome** [comment]
Thanks @kuyagic for opening this issue.
Actually, this has been already fixed in https://github.com/rustic-rs/rustic_core/pull/246 but is not yet included in the nightly build.
We are about to prepare a bugfix-release soon.

**Comment by aawsome** [comment]
Is fixed as we now use rustic_core 0.3.1. This fixed will be included in 0.8.1 (to be released today or tomorrow).
